index.html

```diff
@@ -213,22 +217,33 @@ <h5 class="font-weight-bold text-white mb-lg-4 pb-2 margin-top-mobile">Features<
                     </ul>
                 </div>
 
+
+                <div class="col-lg-3 col-md-3 col-sm-6 col-12 socialicons">
+
+                 <div class="col-lg-3 col-md-3 col-sm-6 col-12 socialicons">
```

Comment:
Line 221 and 223 are same

Reply:
It has some conflits i think .
should i again raise pr